src/app/app.component.spec.ts

```diff
@@ -14,16 +14,20 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have the 'angular-project' title`, () => {
-    const fixture = TestBed.createComponent(AppComponent);
-    const app = fixture.componentInstance;
-    expect(app.title).toEqual('angular-project');
-  });
+  // Keeping this here, will update or remove later.
+
+  // it(`should have the 'angular-project' title`, () => {
+  //   const fixture = TestBed.createComponent(AppComponent);
+  //   const app = fixture.componentInstance;
+  //   expect(app.title).toEqual('angular-project');
+  // });
 
   it('should render title', () => {
     const fixture = TestBed.createComponent(AppComponent);
     fixture.detectChanges();
     const compiled = fixture.nativeElement as HTMLElement;
-    expect(compiled.querySelector('h1')?.textContent).toContain('Hello, angular-project');
+    expect(compiled.querySelector('h1')?.textContent).toContain(
+      'Hello, angular-project'
```

Comment:
might want to change or remove default here

Reply:
I think this is auto generated with the project, not something I did. But I'll check this file again in the future.